providers.tf

```diff
@@ -0,0 +1,4 @@
+terraform {
+  #   required_version  = "~> 0.12.0"
+  required_version = ">= 0.12.0"
```

Comment:
to be really useful we'd want to do something like `0.12.25`.

I think min-version is more useful in projects than modules in general though.

Reply:
true, por que no los dos?